backend/Dockerfile.ocp

```diff
@@ -0,0 +1,40 @@
+ARG BUILD_CONFIGURATION=Release
```

Comment:
This Dockerfile is so that we can build the API with dotnet 5.0.  It is essentially identical to our local build image.  Which is probably better in that we will be developing with the same image.  The main difference is that I have pushed a copy of the images to OpenShift registry to speed up the build process.